Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -278,10 +282,14 @@ open class _ChatMessageContentView<ExtraData: ExtraDataTypes>: _View, ThemeProvi
             errorIndicatorView.setContentCompressionResistancePriority(.streamRequire, for: .horizontal)
             errorIndicatorView.setContentCompressionResistancePriority(.streamRequire, for: .vertical)
 
+            let errorIndicatorContainer = createErrorIndicatorContainer()
+            errorIndicatorContainer.addSubview(errorIndicatorView)
```

Comment:
This creates a new local variable `errorIndicatorContainer`. You probably wanted to assign the value to `self. errorIndicatorContainer`?

Reply:
Yeah, this is done in `createErrorIndicatorContainer` (if the container is `nil` we create it first and then give it to the caller)

Reply:
Ah, yeah ... sorry 👍